### DISCUSSION TITLE
Will you make a tutorial?

### DISCUSSION THREAD

**Discussion by DaniloCannas** [PIut02/ROG-Zephyrus-G14-GA401IV-Hackintosh#7] [Q&A] [answered]
Will you make a tutorial?

**Comment by PIut02** [comment]
About what? There are already many tutorials for using OpenCore, and some BIOS settings and precautions are also written in the readme.

**Reply by DaniloCannas** [reply]
I'm trying to follow the guide, I have a GA401QM (r7 5800hs and 3060), But the usb stucks at the apple logo with loading bar at 0%.
[A.zip](https://github.com/PIut02/ROG-Zephyrus-G14-GA401-Hackintosh/files/12904568/A.zip)

Idk what I' m doing wrong

**Reply by PIut02** [reply, marked as answer]
There are too many errors in this EFI, such as ACPI and Kext. You just need to use the 2021/EFI folder from this repository and remember to clear the NVRAM.
